### PR TITLE
Fix `make build-integration-cli-on-swarm`

### PR DESCRIPTION
**Pull request by AkihiroSuda**
Signed-off-by: Akihiro Suda <suda.akihiro@lab.ntt.co.jp>

**- What I did**

Fix the following failure
```
Building  integration-cli-worker from docker-dev:master
docker run -t -d --name integration-cli-worker-tmp -e DOCKER_GITCOMMIT -e BUILDFLAGS -e DOCKER_INCREMENTAL_BINARY --privileged -v "dockerdev-go-pkg-cache-gopath:/go/pkg" -v "dockerdev-go-pkg-cache-goroot-linux_amd64:/usr/local/go/pkg/linux_amd64" -v "dockerdev-go-pkg-cache-goroot-linux_amd64_netgo:/usr/local/go/pkg/linux_amd64_netgo" docker-dev:master top
dda32f25fbff3f293cee213246cb5b263021f37fbfeb016eaff6706cb9795079
docker exec integration-cli-worker-tmp hack/make.sh build-integration-test-binary dynbinary

---> Making bundle: build-integration-test-binary (in bundles/17.06.0-dev/build-integration-test-binary)
hack/make.sh: line 189: /go/src/github.com/docker/docker/hack/make/build-integration-test-binary: No such file or directory
Makefile:191: recipe for target 'build-integration-cli-on-swarm' failed
make: *** [build-integration-cli-on-swarm] Error 1
```

**- How I did it**

Add hack/make/build-integration-test-binary

**- How to verify it**

```
$ make build-integration-cli-on-swarm
$ hack/integration-cli-on-swarm/integration-cli-on-swarm -replicas XXX
```

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

penguin

relates to https://github.com/moby/moby/issues/34491

### CODE REVIEW

**Review by vdemeester — APPROVED** [review]
LGTM 🐸

File: hack/make/.integration-test-helpers [hunk 1]
```diff
@@ -33,6 +33,10 @@ run_test_integration() {
 }
 
 build_test_suite_binaries() {
+	if [ $DOCKER_INTEGRATION_TESTS_VERIFIED ]; then
```
Comment (dnephin) [inline hack/make/.integration-test-helpers]: nit: this variable read to me like "the tests have run and passed" but I guess it's fine

**Review by dnephin — APPROVED** [review]
LGTM

File: hack/make/.integration-test-helpers (hunk 1 above)
Comment (AkihiroSuda) [inline hack/make/.integration-test-helpers]: The older version of the build system had this variable.
I kept the variable name for compatibility.

**Pull request merged**